UnitsConverter.Fluent.Plugin/UnitsConversionSearchApp.cs

```diff
@@ -109,6 +109,17 @@ public async IAsyncEnumerable<ISearchResult> SearchAsync(SearchRequest searchReq
             string searchedText = searchRequest.SearchedText;
 
             QuantityType quantityType = QuantityType.Undefined;
+
+            if (searchedTag.Equals(UnitsConverterSearchTag) && !string.IsNullOrWhiteSpace(searchedText))
```

Comment:
Just to be sure - when I said without a search tag, I meant without using any search tags.
So users can search something like `5m in seconds`, but I am not sure if full terms are supported, like `seconds` instead of `s`.

Reply:
Does this mean the UnitConversionSearchTag is optional? So the conversion can work with only the searchedText? Please clarify. I have not thought about this.